### PR TITLE
Remove duplicate words

### PR DESCRIPTION
**Pull request by mbolotov**

### CODE REVIEW

File: topics/basics/indexing_and_psi_stubs/file_based_indexes.md [hunk 1]
```diff
@@ -83,4 +83,4 @@ Generally, the word index should be accessed indirectly by using helper methods
 [`FilenameIndex`](upsource:///platform/indexing-api/src/com/intellij/psi/search/FilenameIndex.java) provides a quick way to find all files matching a specific file name.
 
 ### File Type Index
-[`FileTypeIndex`](upsource:///platform/indexing-api/src/com/intellij/psi/search/FileTypeIndex.java) serves a similar goal: it allows to find all files of a particular [`FileType`](upsource:///platform/core-api/src/com/intellij/openapi/fileTypes/FileType.java) quickly.
\ No newline at end of file
+[`FileTypeIndex`](upsource:///platform/indexing-api/src/com/intellij/psi/search/FileTypeIndex.java) serves a similar goal: it allows to find all files of a particular [`FileType`](upsource:///platform/core-api/src/com/intellij/openapi/fileTypes/FileType.java) quickly.
```
Comment (halirutan) [inline topics/basics/indexing_and_psi_stubs/file_based_indexes.md:86]: I can't spot any difference in this sentence. Why is it marked as being different?

File: topics/basics/indexing_and_psi_stubs/file_based_indexes.md [hunk 2]
```diff
@@ -32,7 +32,7 @@ An implementation of a file-based index consists of the following main parts:
 * `getIndexer()` returns the indexer class actually responsible for building a set of key/value pairs based on file content.
 * `getKeyDescriptor()` returns the key descriptor responsible for comparing keys and storing them in a serialized binary format.
 
-   Probably the most commonly used [`KeyDescriptor`](upsource:///platform/util/src/com/intellij/util/io/KeyDescriptor.java) implementation is [`EnumeratorStringDescriptor`](upsource:///platform/util/src/com/intellij/util/io/EnumeratorStringDescriptor.java), which is designed for storing efficiently storing identifiers.
+   Probably the most commonly used [`KeyDescriptor`](upsource:///platform/util/src/com/intellij/util/io/KeyDescriptor.java) implementation is [`EnumeratorStringDescriptor`](upsource:///platform/util/src/com/intellij/util/io/EnumeratorStringDescriptor.java), which is designed for storing identifiers efficiently.
```
Comment (halirutan) [inline topics/basics/indexing_and_psi_stubs/file_based_indexes.md:35]: Good catch. Looks good to me.

File: topics/basics/indexing_and_psi_stubs/file_based_indexes.md (hunk 1 above)
Comment (mbolotov) [reply topics/basics/indexing_and_psi_stubs/file_based_indexes.md:86]: There is some whitespace difference here. If you check this settings, it disappears:
image

I don't know why this whitespace was added/removed. I just used Github page to edit the file.

**Comment by hsz** [comment]
Thanks, Mikhail!

**Pull request merged**